demisto_sdk/commands/common/tests/playbook_test.py

```diff
@@ -75,6 +121,18 @@ def test_is_added_required_fields(self, current_file, answer):
         (NEXT_TASKS_VALID_EXIST_2, True),
     ]
 
+    @pytest.mark.parametrize("current_file, answer", IS_NO_ROLENAME_INPUTS)
+    def test_is_added_required_fields(self, current_file, answer):
+        structure = mock_structure("", current_file)
+        validator = PlaybookValidator(structure)
+        assert validator.is_no_rolename() is answer
+
+    @pytest.mark.parametrize("current_file, answer", IS_CONDITIONAL_INPUTS)
```

Comment:
why do we name these `current_file` and `answer` ? this is confusing..

Reply:
changed to `playbook_json` and `expected_result`